### PR TITLE
RN-952: Switch to bitwarden for pulling .env vars

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue RN-952:

Switch to pulling from bitwarden. We have to use `jq` to parse the responses from bitwarden cli since we can't filter the query.

#### Post merge tasks:
- [ ] Replace `setupGoldMaster.sh` in s3 with latest version
- [ ] Remove Lastpass parameter store entries
- [ ] Update the [monorepo setup docs](https://beyond-essential.slab.com/posts/tupaia-monorepo-setup-v5egpdpq)

### CODE REVIEW

File: scripts/bash/downloadEnvironmentVariables.sh [hunk 1]
```diff
@@ -14,14 +14,23 @@ else
...
+COLLECTION_ID=$(bw get collection "$COLLECTION_PATH" | jq .id)
 
 for PACKAGE in $PACKAGES; do
     ENV_FILE_PATH=${DIR}/../../packages/${PACKAGE}/.env
 
-    # checkout deployment specific env vars, or dev as fallback, temporarily redirecting stderr
-    lpass show --notes "${FOLDER}/${PACKAGE}.${DEPLOYMENT_NAME}.env" 2> /dev/null > ${ENV_FILE_PATH} \
-        || lpass show --notes "${FOLDER}/${PACKAGE}.dev.env" > ${ENV_FILE_PATH}
+    # checkout deployment specific env vars, or dev as fallback
+    DEPLOYEMNT_ENV_VARS=$(bw list items --search ${PACKAGE}.${DEPLOYMENT_NAME}.env | jq --raw-output "map(select(.collectionIds[] | contains ($COLLECTION_ID))) | .[] .notes")
```
Comment (alexd-bes) [inline scripts/bash/downloadEnvironmentVariables.sh:27]:
```suggestion
    DEPLOYMENT_ENV_VARS=$(bw list items --search ${PACKAGE}.${DEPLOYMENT_NAME}.env | jq --raw-output "map(select(.collectionIds[] | contains ($COLLECTION_ID))) | .[] .notes")
```

Comment (alexd-bes) [inline scripts/bash/downloadEnvironmentVariables.sh:27]:
```suggestion
    DEPLOYMENT_ENV_VARS=$(bw list items --search ${PACKAGE}.${DEPLOYMENT_NAME}.env | jq --raw-output "map(select(.collectionIds[] | contains ($COLLECTION_ID))) | .[] .notes")
```

**Review by alexd-bes — APPROVED** [review]
Nice, this is awesome! One more post-merge task: update the [monorepo setup docs](https://beyond-essential.slab.com/posts/tupaia-monorepo-setup-v5egpdpq)

File: packages/devops/scripts/deployment/buildDeployablePackages.sh [hunk 2]
```diff
@@ -20,9 +20,9 @@ yarn install --immutable
 yarn build:internal-dependencies
 
 # Inject environment variables from LastPass
```
Comment (jaskfla) [inline packages/devops/scripts/deployment/buildDeployablePackages.sh:22]:
```suggestion
# Inject environment variables from Bitwarden
```

**Pull request merged**